x/concentrated-liquidity/fees_test.go

```diff
@@ -888,39 +954,60 @@ func (s *KeeperTestSuite) TestUpdateFeeAccumulatorPosition() {
 			s.SetupTest()
 
 			// Setup two cl pools
-			poolOne := s.PrepareConcentratedPool()
+			pool := s.PrepareConcentratedPool()
```

Comment:
This says setup two cl pools but I only see one

Reply:
removed obsolete comment